Delicate-Butterflies/Bangazon-Workforce
# Mb as navbar

Related Issue: #1 

Description: Created navbar and other pug templates 

Chai/Mocha Tests File: NA

PR Testing Steps: 
- run app.js
- check the home page 
- check /employees route 

Check list before you make a PR:
